lib/utils.js

```diff
@@ -490,6 +501,12 @@ api.verifyCapabilityChain = async ({
       throw error;
     }
 
+    // if root.expires is undefined then Date.parse returns NaN which is falsey
+    const rootExpirationDate = Date.parse(root.expires);
+    if(rootExpirationDate && currentDate > rootExpirationDate) {
```

Comment:
There's an epoch edge case bug here.  Date.parse returns an int.  So root.expires='1970' would result in 0 and skipping the test.  That would make people checking for validity with a currentDate in 1969 sad.  Should perhaps do `new Date(Date.parse(root.expires))` to get a Date object, but then need to deal with the potential 'Invalid Date' result for an invalid root.expires.  Dates are hard.  While this example is unlikely, there may be tricky data that could slip through the checks.

Reply:
@davidlehn do we have code from jsonld-sigs or elsewhere that addresses these concerns?  We check for expired dates like this in other libraries right?

Reply:
@davidlehn does this implementation from jsonld-signatures address your concerns? https://github.com/digitalbazaar/jsonld-signatures/blob/531058bf1af7101bd261f540e4b875f3e5101076/lib/purposes/ProofPurpose.js#L49-L58

Reply:
Attempted to address: 585228e46ef73c99ce7a2c436000ac9a20714e04

Reply:
Looks more robust now.  It's a challenge to consider odd edge cases.  I imagine this works for regular use cases.

Reply:
Yes, the test suite still passes for regular use cases and I just add a couple more test related to Unix epoch edge case.